WordPress/src/main/java/org/wordpress/android/support/ZendeskHelper.kt

```diff
@@ -367,7 +369,9 @@ private fun buildZendeskCustomFields(
             CustomField(TicketFieldIds.currentSite, currentSiteInformation),
             CustomField(TicketFieldIds.deviceFreeSpace, DeviceUtils.getTotalAvailableMemorySize()),
             CustomField(TicketFieldIds.logs, AppLog.toPlainText(context)),
-            CustomField(TicketFieldIds.networkInformation, getNetworkInformation(context))
+            CustomField(TicketFieldIds.networkInformation, getNetworkInformation(context)),
+            CustomField(TicketFieldIds.appLanguage, Locale.getDefault().getLanguage()),
```

Comment:
The `Locale.getDefault().getLanguage()` parameter can be `Locale.getDefault().language`.  It's effectively the same thing, but `Locale.getDefault().language` lets you harness the true power of Kotlin synthetic properties.  I'm fine with merging this either way.